x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -326,11 +323,6 @@ export type CaseFullExternalService = rt.TypeOf<typeof CaseFullExternalServiceRt
 export type CaseSettings = rt.TypeOf<typeof SettingsRt>;
 export type ExternalServiceResponse = rt.TypeOf<typeof ExternalServiceResponseRt>;
 
-export type ESCaseAttributes = Omit<CaseAttributes, 'connector'> & { connector: ESCaseConnector };
-export type ESCasePatchRequest = Omit<CasePatchRequest, 'connector'> & {
-  connector?: ESCaseConnector;
-};
-
```

Comment:
These have been moved into the cases service because they are isolated to the service layer. Other parts of cases no longer need to reference the types. Instead they will always reference the `CaseAttributes` or equivalent type.